src/ClientData/include/ClientData/DataManager.h

```diff
@@ -115,6 +120,7 @@ class DataManager final {
 
   uint32_t selected_thread_id_ = orbit_base::kInvalidThreadId;
   const orbit_client_protos::TimerInfo* selected_timer_ = nullptr;
+  std::optional<orbit_client_data::ThreadStateSliceInfo> selected_thread_state_slice_{std::nullopt};
```

Comment:
The `{std::nullopt}` is not needed since that's what the default constructor of `std::optional` does anyway.